cloudwatch_logs_common/src/utils/log_file_manager.cpp

```diff
@@ -15,32 +15,38 @@
 
 
 
-#include <memory>
-#include <iostream>
+#include <chrono>
 #include <fstream>
+#include <iostream>
+#include <memory>
+#include <queue>
+#include <tuple>
+
 #include "cloudwatch_logs_common/utils/log_file_manager.h"
 #include "file_management/file_upload/file_manager_strategy.h"
 #include <aws/core/utils/json/JsonSerializer.h>
 #include <aws/core/utils/logging/LogMacros.h>
 #include <cloudwatch_logs_common/definitions/definitions.h>
 
+#define HOURS24 86400000
```

Comment:
```suggestion
const long ONE_DAY_IN_SEC = 86400000;
```

Or maybe

```
const long ONE_DAY_IN_SEC = 24 * 60 * 60;
```

Better to use a typed const and define your units